contract.go

```diff
@@ -241,6 +251,10 @@ func (e *ContractExecutor) Execute(snapshot *avl.Tree, id AccountID, round *Roun
 		return errors.Wrapf(ErrContractFunctionNotFound, `fn "_contract_%s" does not exist`, name)
 	}
 
+	if vm.FunctionCode[entry].NumParams != 0 {
```

Comment:
Why must the entry function not have parameters?

Reply:
That's our contract ABI:

https://github.com/perlin-network/smart-contract-rs/blob/master/smart-contract-macros/src/lib.rs#L168
https://github.com/perlin-network/smart-contract-rs/blob/master/smart-contract-macros/src/lib.rs#L185